apteryx-xml.h

```diff
@@ -181,6 +183,7 @@ GList *sch_parm_removes (sch_xml_to_gnode_parms parms);
 GList *sch_parm_creates (sch_xml_to_gnode_parms parms);
 GList *sch_parm_replaces (sch_xml_to_gnode_parms parms);
 void sch_parm_free (sch_xml_to_gnode_parms parms);
+sch_node *sch_node_and_namespace_get_child (xmlNs *ns, sch_node * parent, const char *child);
```

Comment:
sch_node *sch_node_child_ns (sch_node * parent, const char *child, const char *namespace);

and place it below sch_node_child above

this fn should be part of the sch_node abstraction and should not use anything from libxml2 ... i would prefer to not require prefix ... 

if you really have to you could make the prototype

sch_node *sch_node_child_ns (sch_node * parent, const char *child, const char *namespace, const char *prefix);

but make the prefix an optional parameter

Reply:
The function has been renamed and no longer passes a xmlNs pointer. The position of the declaration in apteryx-xml.h has also been fixed up.